match/coms/service_coms_handler.go

```diff
@@ -0,0 +1,30 @@
+package coms
+
+import (
+	"fmt"
+	"github.com/TempleEight/spec-golang/match/utils"
+	"net/http"
+)
+
+// Handler maintaints the list of services and their associated hostnames
+type Handler struct {
+	Services map[string]string
+}
+
+// Init sets up the Handler object with a list of services from the config
+func (coms *Handler) Init(config *utils.Config) {
+	coms.Services = config.Services
+}
+
+// CheckUser makes requests to the target service to check if a userId exists.
+func (coms *Handler) CheckUser(service string, userID int) (bool, error) {
```

Comment:
Why are we passing the service name in here? Surely `CheckUser` can only be to one service

Reply:
It also avoids the programmer of `matches.go` from knowing the service name